pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
 env =
-    SKETCHBOOK_DIR=tests-skecthbook
+    SKETCHBOOK_DIR=tests-sketchbook
```

Comment:
I have fixed this typo, but after rebasing my branch noticed that you did that as well :P

Reply:
No worries...